src/routes/news/components/TunewsDetailsFooter.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import React from 'react'
+import styled from 'styled-components'
+import { TFunction } from 'i18next'
+import type Moment from 'moment'
+
+const Footer = styled.footer`
+  display: flex;
+  width: 96%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  background-color: ${({ theme }) => (theme.colors.secondaryAccentColor)};
+  padding: 0 2% 5px;
+  color: ${({ theme }) => (theme.colors.backgroundColor)};
+  border-radius: 24px;
+`
+
+const StyledContainer = styled.div`
+  padding: 5px 10px 0;
+`
+
+const StyledLink = styled.a`
+  color: ${({ theme }) => (theme.colors.backgroundColor)};
+  text-decoration: underline;
+`
+
+type PropsType = {|
+  eNewsNumber: string,
+  date: Moment,
+  language: string,
+  t: TFunction
+|}
+
+class TunewsDetailsFooter extends React.PureComponent<PropsType> {
+  render () {
+    const { eNewsNumber, date, language, t } = this.props
+    date.locale(language)
+    const timestamp = date.format('LL')
+
+    return (
+      <Footer>
+        <StyledContainer>{t('eNewsNo')}: {eNewsNumber}</StyledContainer>
+        <StyledContainer>
+          <StyledLink href='http://www.tunews.de' target='_blank'>
+            Tunews International
```

Comment:
```suggestion
            tünews INTERNATIONAL
```

@svenseeberg what is the correct name of tünews?

Reply:
Fixed as the suggestion